mmv1/products/datastream/Stream.yaml

```diff
@@ -107,6 +107,23 @@ examples:
       stream_id: 'my-stream'
       source_connection_profile_id: 'source-profile'
       destination_connection_profile_id: 'destination-profile'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'datastream_stream_sql_server'
+    min_version: beta
```

Comment:
```suggestion
```

Reply:
Removed in https://github.com/GoogleCloudPlatform/magic-modules/commit/939b4f1e8962c3ac939f24d2fc98170f48bdc4d7